wgengine/magicsock/legacy.go

```diff
@@ -129,6 +164,57 @@ func (c *Conn) sendAddrSet(b []byte, as *addrSet) error {
 	return ret
 }
 
+// peerFromPacketLocked extracts returns the addrSet for the peer who sent
+// packet, if derivable.
+func (c *Conn) peerFromPacketLocked(packet []byte) *addrSet {
+	if len(packet) < 4 {
+		return nil
+	}
+	msgType := binary.LittleEndian.Uint32(packet[:4])
+	if msgType != device.MessageInitiationType {
```

Comment:
We've copied a bunch out of wireguard-go, may as well copy this constant too.  It can't change, since it is part of the wire protocol.